libdnf/base/goal.cpp

```diff
@@ -132,13 +130,13 @@ void Goal::add_rpm_remove(const libdnf::rpm::Package & rpm_package) {
 }
 
 void Goal::add_rpm_remove(const libdnf::rpm::PackageSet & package_set) {
-    for (auto package_id : *package_set.pImpl) {
+    for (auto package_id : *package_set.p_impl) {
         p_impl->remove_rpm_ids.push_back(package_id.id);
     }
 }
 
 void Goal::add_rpm_remove(const libdnf::rpm::SolvQuery & query) {
```

Comment:
Minor - I suggest that the whole function could be removed

Reply:
Thanks. Removed.